geometry_msgs/Pose goal_pose
bool is_goal_relative
---
bool answer